web/js/layers/options.js

```diff
@@ -147,8 +147,7 @@ export function layersOptions(config, models, layer) {
         }
       })
       .on('slide', function () {
-        models.layers.setOpacity(layer.id, parseFloat($(this)
-          .val()));
+        models.layers.setOpacity(layer.id, $(this).val());
```

Comment:
Why did you remove the `parseFloat` here?

Reply:
`parseFloat` was affecting the click dragging of the slider. There were performance issues / bugginess. Removing it fixes these issues. The step, min and max are defined so I don't see why it is needed.